src/lightning_app/frontend/stream_lit.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, render_fn: Callable) -> None:
 
     def start_server(self, host: str, port: int) -> None:
         env = os.environ.copy()
+        is_localhost = "LIGHTNING_APP_STATE_URL" not in env
```

Comment:
FYI we have a utility if you want to use that: `lightning_app.utilities.cloud.is_running_in_cloud()`

Reply:
thankyou @awaelchli, changed!